pynxtools_xps/reader.py

```diff
@@ -140,22 +145,25 @@ def fill_data_group(key, key_part, entries_values, config_dict, template):
             count += 1
             template[f"{root}@default"] = entry
 
-        data_field_key = key.replace("entry", entry)
-        data_group_key = data_field_key.rsplit("/data", 1)[0]
+        modified_key = key.replace("entry", entry)
```

Comment:
This might be safer when there is the string `entry` somewhere else in the path

```suggestion
        modified_key = key.replace("/ENTRY[entry]/", f"/ENTRY[{entry}]/")
```

Reply:
Or is it just the hdf path here, then something like this
```suggestion
        modified_key = key.replace("/entry/", f"/{entry}/")
```

Reply:
Very good idea, I implemented it everywhere, also in #5 👍